cmd/unleash.go

```diff
@@ -45,6 +45,8 @@ func newUnleashCmd() *unleashCmd {
 		Long:    `Unleashes the gremlins and performs mutation testing on a Go module.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			log.Infoln("Starting...")
+			runDir, _ := os.Getwd()
```

Comment:
Is it correct not to check the error here?

Reply:
hum...actually it would be better to check it, let me fix it :)